### DISCUSSION TITLE
Can't set resolution to 640x480

### DISCUSSION THREAD

**Discussion by mitrivaker** [ppy/osu#20527] [Q&A]
### Type

Game behaviour

### Bug description

Hello,

I didn't find any the option to set the game's resolution 640x480 so I modified the `SizeFullscreen` value to `640x480` in the `framework.ini` file located in `%AppData%/osu/`

It works but when I open the settings panel in game, it changes the value to `9999x9999` and I have to do it again.

### Screenshots or videos

https://user-images.githubusercontent.com/114605874/192848514-e2675f51-809e-4735-8d7f-47caf3f4295c.mp4

### Version

2022.923.1-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9667338/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9667339/updater.log)
[database.log](https://github.com/ppy/osu/files/9667340/database.log)
[network.log](https://github.com/ppy/osu/files/9667341/network.log)
[performance.log](https://github.com/ppy/osu/files/9667342/performance.log)

**Comment by smoogipoo** [comment]
What are you trying to achieve? Change your screen's resolution to 640x480?

**Reply by peppy** [reply]
I think what's going on is that 640x480 isn't being shown in the valid resolutions dropdown, even though it is supported by the GPU. I've seen this come up in some other cases (I think "overclocked" refresh rates also have the same issue).

**Reply by mitrivaker** [reply]
Yes, my goal is to be able to play at 640x480 so I can get a higher refresh rate on my monitor, I do the same on osu! Stable and it works.
